src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -74,10 +73,6 @@ export default {
 			type: Boolean,
 			default: false
 		},
-		userBalance: {
```

Comment:
This can be removed from the component where it's been used in `LoanFinding`

Reply:
I think is removed from there, https://github.com/kiva/ui/blob/25593925f87498e1327eb641fe913c33da767931/src/components/LoanFinding/QuickFiltersSection.vue#L77

Reply:
I'm talking about in `LoanFinding`:

https://github.com/kiva/ui/blob/53434de804224add9d7833860d83b217776e3135/src/pages/LoanFinding/LoanFinding.vue#L33

Reply:
The component no longer needs the balance provided